webpack.config.js

```diff
@@ -20,16 +20,19 @@ dotenv.config({path: '.env.default'});
  * @returns {object}
  */
 module.exports = (env = {NODE_ENV: process.env.NODE_ENV || 'production'}) => ({
-  entry:
-    env && env.NODE_ENV === 'development'
-      ? `${path.resolve(__dirname)}/packages/webex/src/index.js`
-      : './packages/webex',
+  entry: {
+    webex:
+      env && env.NODE_ENV === 'development'
+        ? `${path.resolve(__dirname)}/packages/webex/src/index.js`
+        : './packages/webex',
+    meetings: `${path.resolve(__dirname)}/packages/webex/meetings/src/index.js`,
```

Comment:
can we move the meetings folder inside webex/src , rather then webex/ meeting/src 

and remove the stats.json